client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -83,6 +90,11 @@ const DataFilesSearchbar = ({ api, scheme, system, className, publicData }) => {
           autoComplete="off"
           disabled={disabled}
         />
+        {hasQuery && (
+          <div aria-label="Summary of Search Results" styleName="results">
+            {resultCount} Results Found for: &quot;<span>{hasQuery}</span>&quot;
```

Comment:
__Does Not Match Design, But Maybe Quotes are Good__

```suggestion
            {resultCount} Results Found for <span>{hasQuery}</span>
```

The [design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/02e5a107-a2d7-45ae-a2fa-2fcb9aabe2bc/specs/) does **not** use colon **nor** quote marks. Functionally though, the quote marks are accurate, because search fuzziness was removed (#399). I'll confirm with Design.

Reply:
FYI, the [task](https://jira.tacc.utexas.edu/browse/FP-563) does use quote marks, so would be good to clarify

Reply:
If we do not have an answer by end of the day, then please remove quotes.

Reply:
Great catch, @rstijerina. That discrepancy is _probably_ my fault, though. I may have made a mistake when I wrote that—I have a habit of quoting things when I type (notice the `and "FP-563" in comments` in the task desc.). When [I coded the text](https://github.com/TACC/Core-Portal/pull/96/files#diff-6f2b9fe9211b0c45ea1978ca2ceb54012b70323ef8260a351033d0de81a878f2R4), I did so without quotes.

P.S. The task "Activity" does not show an edit where I added the quotes in after initially not having them. Such an edit would suggest having _fixed_ requirements.

Reply:
Removing quotes.